gradle.properties

```diff
@@ -0,0 +1,6 @@
+org.gradle.jvmargs=-Dfile.encoding=UTF-8 -Xmx2g -XX:MaxMetaspaceSize=1g
+org.gradle.parallel=true
+org.gradle.caching=true
+# Note that Kotlin/Multiplatform publishing is incompatible with CC, and should be disabled
+org.gradle.unsafe.configuration-cache=true
```

Comment:
The comment says that we should disable it, but then it is enabled anyway.

Do you mean that it could stay on for development, but we have to remember to turn it off whenever we execute publishing tasks?